### PR TITLE
[Token Info] Add Token Chart Section

### PR DESCRIPTION
**Pull request by JoseRFelix**
## What is the purpose of the change

This PR adds the initial section for token chart. Due to a change in priorities, I won't proceed with the token info page at this time. Still, we should consider merging this update, as it incorporates a gradient design for token charts, which will also enhance the CL pools chart.

image

### ClickUp Task

[ClickUp Task URL](https://app.clickup.com/t/86a0pv1q1)

## Brief Changelog

- Add token info UI config
- Add Gradient to token pair chart

## Testing and Verifying

- [ ] CL Pools chart should include a gradient

### CODE REVIEW

File: packages/web/components/chart/token-pair-historical.tsx [hunk 1]
```diff
@@ -80,15 +90,40 @@ const TokenPairHistoricalChart: FunctionComponent<{
...
+                data={data}
+                xAccessor={(d: { time: number; close: number }) => d?.time}
+                yAccessor={(d: { time: number; close: number }) => d?.close}
+                fillOpacity={0.4}
+                curve={curveNatural}
+                fill="url(#gradient)"
+              />
+              <LinearGradient
+                id="gradient"
+                from="#C41BFF"
```
Comment (MaxMillington) [inline packages/web/components/chart/token-pair-historical.tsx:107]: can these be tailwind variables?

File: packages/stores/src/ui-config/asset-info-config.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+import {
+  PriceRange,
+  QueriesExternalStore,
+  TimeFrame,
+} from "../queries-external";
+
+const INITIAL_ZOOM = 1.05;
+const ZOOM_STEP = 0.05;
+
+export class ObservableAssetInfoConfig {
```
Comment (MaxMillington) [inline packages/stores/src/ui-config/asset-info-config.ts:15]: Why is a ui-config needed?

Comment (jonator) [reply packages/stores/src/ui-config/asset-info-config.ts:15]: I think we decided on this in the TDD.

Though we still need to write a spec for it.

See comment for rationale

https://docs.google.com/document/d/1zw47fYdvLcEeJvRZiQqDfglSvpBLf0KDKyJ7Rw0UN3k/edit?disco=AAAA4kn39Mo

File: packages/web/components/chart/token-pair-historical.tsx (hunk 1 above)
Comment (jonator) [reply packages/web/components/chart/token-pair-historical.tsx:107]: FYI @JoseRFelix it's possible to import the tailwind config

Comment (JoseRFelix) [reply packages/web/components/chart/token-pair-historical.tsx]: Thanks for the suggestion! Added the variables on my new commit

**Comment by jonator** [comment]
Great work on the gradient

**Review by MaxMillington — APPROVED** [review]

**Pull request merged**